services/cd-service/pkg/repository/transformer_test.go

```diff
@@ -1458,26 +1459,22 @@ func TestTransformer(t *testing.T) {
 		},
 		{
 			Name:         "Lock env AND app and then Deploy and unlock both locks",
-			Transformers: makeTransformersDoubleLock(api.LockBehavior_Queue, true),
+			Transformers: makeTransformersDoubleLock(api.LockBehavior_Record, true),
 			Test: func(t *testing.T, s *State) {
 				// check that the state reads the correct versions
 				i, err := s.GetEnvironmentApplicationVersion("production", "test")
 				if err != nil {
 					t.Fatal(err)
 				}
-				if i == nil {
-					t.Errorf("unexpected version: expected 1, actual nil")
-				} else {
-					if *i != 1 {
-						t.Errorf("unexpected version: expected 1, actual %d", *i)
-					}
+				if i != nil {
+					t.Errorf("unexpected version: unexpected nil")
```

Comment:
Can you explain the error message, please? 

```suggestion
					t.Errorf("unexpected version %d: expected: nil", i)
```